### DISCUSSION TITLE
Dshidmini - igfilter Fails to Install (Solved)

### DISCUSSION THREAD

**Discussion by veersw** [nefarius/DsHidMini#283] [Support]
After right-clicking on igfiter.inf and allowing through windows security/install prompts, it fails to install with the following error.
Had anyone else run into this / found a way around it?
Thanks in advance.
![image](https://user-images.githubusercontent.com/127800084/224849232-4736b951-35e3-48d8-b9aa-bb01b219226c.png)

**Comment by veersw** [comment]
Tried to install incorrect architecture

**Reply by wfllama** [reply]
I'm having the same issue with the igfilter.inf file not installing, would you mind telling me how you fixed it?

**Reply by nefarius** [reply]
https://vigem.org/research/How-to-check-architecture/
